cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -0,0 +1,1324 @@
+        .append('<th>Action</th>')  
+        .appendTo(thead);
+
+    // Fetch data from "Payroll Configuration"
+    frappe.call({
+        method: "frappe.client.get",
+        args: {
+            doctype: "Payroll Configuration",
+            fields: ["salary_component_configuration"]
+        },
+        callback: function(res) {
+            if (res.message && res.message.salary_component_configuration) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)